src/client.rs

```diff
@@ -874,8 +874,21 @@ where
                 Err(e) => return (delegation.terminating(), Err(e)),
             };
 
+            /////////////////////////////////////////
+            // TUF-1.0.9 §5.4:
+            //
+            //     Download the top-level targets metadata file, up to either the number of bytes
+            //     specified in the snapshot metadata file, or some Z number of bytes. The value
```

Comment:
I wish the spec was more explicit about Z being a max length for the downloaded file, but that's a fix for elsewhere.